packages/foreman/nodejs-loose-envify/nodejs-loose-envify.spec

```diff
@@ -0,0 +1,79 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name loose-envify
+
+Name: %{?scl_prefix}nodejs-loose-envify
+Version: 1.4.0
+Release: 1%{?dist}
+Summary: Fast (and loose) selective `process
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/zertosh/loose-envify
+Source0: https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz
+Source1: https://registry.npmjs.org/loose-envify/-/loose-envify-1.4.0.tgz
+Source2: nodejs-loose-envify-%{version}-registry.npmjs.org.tgz
+BuildRequires: %{?scl_prefix_nodejs}npm
+%if 0%{!?scl:1}
+BuildRequires: nodejs-packaging
+%endif
+BuildArch: noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
+Provides: bundled(npm(js-tokens)) = 4.0.0
```

Comment:
We have js-tokens 4.0.0 packaged, no need to bundle it.

Reply:
Can we partially-bundle a package?  We still need to bundle loose-envify

Reply:
This is the loose-envify package, so no, once we unbundle js-tokens, this package is "simple".